core/frontend/src/tile/GltfReader.ts

```diff
@@ -424,11 +423,13 @@ export abstract class GltfReader {
     return id ?? extractId(materialJson.pbrMetallicRoughness?.baseColorTexture?.index);
   }
 
+
   protected createDisplayParams(materialJson: any, hasBakedLighting: boolean): DisplayParams | undefined {
     const textureId = this.extractTextureId(materialJson);
     const textureMapping = undefined !== textureId ? this.findTextureMapping(textureId) : undefined;
     const color = this.colorFromMaterial(materialJson);
-    return new DisplayParams(DisplayParams.Type.Mesh, color, color, 1, LinePixels.Solid, FillFlags.Always, undefined, undefined, hasBakedLighting, textureMapping);
+    const material = hasBakedLighting ? undefined : this._system.createMaterial(RenderMaterial.Params.defaults, this._iModel);

```

Comment:
If mesh has no material we use defaults - necessary to create materials from defaults here?

Reply:
Yes correct - this was an unnecessary change I tried when I couldn't get lighting to work - I backed it out.